File: UCFPT/21highschoolonlinedec/contest/solutions/piano/piano.py
# piano.py by Jacob Steinebronn

# The number of test cases
nt = (int)(input().strip())

for tn in range(nt):
    # Get the input string
    word = input().strip()
    # We can just use the left hand for everything!
    print("L"*len(word))

File: UCFPT/21highschoolonlinedec/contest/solutions/piano/piano.cpp
// 2021 UCF HSPT Online - Piano
// Author: Josh Delagdo

#include <bits/stdc++.h>

using namespace std;

int main() {
  // Read in the number of tests
  int numberTests;
  cin >> numberTests;

  // Go through every test
  for (int test=0; test<numberTests; test++) {
    // Read in the song
    string song;
    cin >> song;

    // Length of the song
    int length = song.length();

    // Since we have the ability to play every node with one hand,
    //   we can play the entire song without ever crossing hands by only
    //   playing with our right hand.
    string output = "";
    for (int i=0; i<length; i++) {
      output += "R";
    }

    // Output the answer
    cout << output << endl;
  }

  return 0;
}